### DISCUSSION TITLE
how get back the taskbar Arrow ??

### DISCUSSION THREAD

**Discussion by HorrorBest** [farag2/Sophia-Script-for-Windows#259] [Q&A] [answered]
hey, I ran Sophia and It was fine until I realized that it removed the thing "idk what to call it" its the small up arrow in the taskbar that has the apps in it" the one in the pic, What function should I call to get it back ??? 
![image](https://user-images.githubusercontent.com/75090757/135841433-ff2028d9-a3d3-40b3-a2c6-936cd18b9d28.png)

**Comment by farag2** [comment, marked as answer]
<https://github.com/farag2/Sophia-Script-for-Windows/blob/b916de13a99fd4fae59d59480fb3ba38d60f3e63/Sophia%20Script/Sophia%20Script%20for%20Windows%2010/Sophia.ps1#L426>

```powershell
sophia.ps1 -Functions "NotificationAreaIcons -Hide"
```

**Reply by HorrorBest** [reply]
**thx <3 that was fast**

**Reply by farag2** [reply]
As always. :)
